### PR TITLE
Append file references

### LINKED ISSUE #869: Fix babel-gettext-extractor to collect file refs
When you have a duplicate string, babel-gettext-extractor is currently ignoring the references. 

Whilst this needs to be fixed in babel-gettext-extractor it makes sense to track it here.

### PR DESCRIPTION
**Pull request by muffinresearch**
Fixes #869

* Update babel-gettext-extractor to version with file reference appending patch.
* Extract to capture file references in the pot
* Merge the changes into the locale files.

### CODE REVIEW

File: locale/sl/LC_MESSAGES/disco.po [hunk 1]
```diff
@@ -1,17 +1,18 @@
...
 "PO-Revision-Date: 2016-08-02 17:58+0000\n"
 "Last-Translator: Lan Glad <upwinxp@gmail.com>\n"
 "Language-Team: none\n"
 "Language: sl\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"Plural-Forms: nplurals=4; plural=(n%100==1 ? 0 : n%100==2 ? 1 : n%100==3 || n%100==4 ? 2 : 3);\n"
+"Plural-Forms: nplurals=4; plural=(n%100==1 ? 0 : n%100==2 ? 1 : n%100==3 || n"
+"%100==4 ? 2 : 3);\n"
```
Comment (kumar303) [inline locale/sl/LC_MESSAGES/disco.po]: is it ok that this line was broken?

File: locale/uk/LC_MESSAGES/disco.po [hunk 2]
```diff
@@ -1,17 +1,18 @@
...
 "PO-Revision-Date: 2016-08-02 08:43+0000\n"
 "Last-Translator: Artem Polivanchuk <a.polivanchuk@outlook.com>\n"
 "Language-Team: none\n"
 "Language: uk\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"Plural-Forms: nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && n%10<=4 && (n%100<10 || n%100>=20) ? 1 : 2);\n"
+"Plural-Forms: nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && n"
+"%10<=4 && (n%100<10 || n%100>=20) ? 1 : 2);\n"
```
Comment (kumar303) [inline locale/uk/LC_MESSAGES/disco.po]: here's another one

File: locale/sl/LC_MESSAGES/disco.po (hunk 1 above)
Comment (muffinresearch) [inline locale/sl/LC_MESSAGES/disco.po]: I think so that's just msgmerge doing its thing.

Comment (kumar303) [inline locale/sl/LC_MESSAGES/disco.po]: ok

File: locale/uk/LC_MESSAGES/disco.po (hunk 2 above)
Comment (kumar303) [inline locale/uk/LC_MESSAGES/disco.po]: looks like this is fine

**Comment by kumar303** [comment]
r+

**Pull request merged**